packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -364,7 +361,10 @@ export default defineComponent({
         return
       }
       const userShares = this.sharesTree[sharePathParentOrCurrent]?.filter((s) =>
-        ShareTypes.containsAnyValue(ShareTypes.individuals, [s.shareType])
+        ShareTypes.containsAnyValue(
+          [...ShareTypes.individuals, ...ShareTypes.unauthenticated],
```

Comment:
didn't check further in the code, but assigning link shares into a const named `userShares` seems hacky.

Reply:
Thx for the hint, I renamed the variable. I don't see any reason why we shouldn't include link shares here.

Reply:
👍